datastore/load_test.go

```diff
@@ -1290,6 +1290,23 @@ func TestLoadNull(t *testing.T) {
 	}
 }
 
+func TestLoadNilInterface(t *testing.T) {
+	type WithAny struct {
+		AnyField interface{}
+	}
+
+	withAny1 := &WithAny{}
+	err := loadEntityProto(withAny1, &pb.Entity{
+		Key: keyToProto(testKey0),
+		Properties: map[string]*pb.Value{
+			"AnyField": {ValueType: &pb.Value_NullValue{}},
+		},
+	})
+	if err != nil {
+		t.Fatalf("got: %v, want: nil", err)
+	}
```

Comment:
Maybe an additional assert check to ensure the value loaded is actually zero?

Reply:
Added